recipes/snappy/bld.bat

```diff
@@ -0,0 +1,12 @@
+cd %SRC_DIR%
+git clone https://github.com/kmanley/snappy-msvc
```

Comment:
> PS: Maybe we should just download those files and ship with the recipe. Just adding a comment about that repo.

In this case, the two important files are the ``sln`` and ``vcproj``. ``stdint`` can be got from the ```msinttypes``` package (https://github.com/conda-forge/msinttypes-feedstock). @groutr - are you happy to just copy those files into the recipe?